src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -90,6 +96,8 @@ public void Loaded(ViewLoadedParams viewLoadedParams)
                 DependencyView.DependencyRegen(viewLoadedParams.CurrentWorkspaceModel as WorkspaceModel);
             };
 
+            viewLoadedParams.OnExtensionTabClosed += OnCloseExtension;
```

Comment:
I would run the self serve ASAP to see if this causes a memory leak during testing.

Reply:
Yes, started the self-serve